spec/models/spree/payment_original_spec.rb

```diff
@@ -0,0 +1,821 @@
+require 'spec_helper'
```

Comment:
nice! I assume the spec will be called payment_spec in the end, right?

Reply:
Yep, I'm gradually moving specs to this one and I'll rename when done.